force-app/main/default/lwc/staffAvailability/staffAvailability.js

```diff
@@ -86,7 +86,7 @@ export default class StaffAvailability extends LightningElement {
 		this.template.querySelector("c-app-spinner").displaySpinner(true);
 		getAssignmentData({contactId: this.recordId? this.recordId:null}).then(result => {
 			this.data = result;
-			this.staffStatus = getFieldValue(result.contact, 'Status__c', this.namespace);
+			this.staffStatus = getFieldValue(result.contact, 'StatusLabel', '');
```

Comment:
could do:

this.staffStatus = result.contact.StatusLabel

NOTE: will have to be tested in the package to figure out whether it requires a namespace or now